.storybook/webpack.config.js

```diff
@@ -11,6 +11,7 @@ function addWebpackAliases(config) {
         "@css": path.resolve(__dirname, "../packages/css/src"),
         "@components": path.resolve(__dirname, "../packages/components/src"),
         "@orbit-ui/components": path.resolve(__dirname, "../packages/components/src"),
+        "@sharegate/orbit-ui": path.resolve(__dirname, "../packages/bundle/src"),
         "@experimental": path.resolve(__dirname, "../packages/experimental/src"),
         "@stories/components": path.resolve(__dirname, "./components"),
         "@stories/mdx": path.resolve(__dirname, "./mdx"),
```

Comment:
Do you actually still need those aliases?

Maybe those related to CSS files but those related to JS files (ts, tsx, ...) should be managed by TSC, not Webpack

Reply:
Ah it's for SB, I guess it's out of scope of TSC :(

Reply:
Yeah, its related to the new-ish Experimental package. I noticed that storybook was failing if we started it from a "fresh" install. It worked if the bundle was built once.